config/repos.yaml

```diff
@@ -93,6 +93,13 @@ theforeman/foreman-selinux:
   redmine_required: true
 theforeman/foreman_setup:
   redmine: foreman_setup
+theforeman/foreman_statistics:
+  redmine: statistics
+  redmine_required: true
+  branch_labels:
+    .+-stable: Stable branch
+  path_labels:
+    webpack: UI
```

Comment:
Note that the processor doesn't create these labels. It's up to you to create them.

Reply:
Thanks for the heads up, noted, but I suspected that being the case, so I got them created: https://github.com/theforeman/foreman_statistics/labels